util/fusermount.c

```diff
@@ -790,10 +820,16 @@ static int do_mount(const char *mnt, char **typep, mode_t rootmode,
 						flags |= flag;
 					else
 						flags  &= ~flag;
-				} else {
```

Comment:
Not sure about this. Is passing unknown options to the kernel really a significant attack surface? It has the drawback that it requires updating libfuse when the kernel starts supporting a new option.